var/spack/repos/builtin/packages/dftd4/package.py

```diff
@@ -10,11 +10,13 @@ class Dftd4(MesonPackage):
     """Generally Applicable Atomic-Charge Dependent London Dispersion Correction"""
 
     homepage = "https://www.chemie.uni-bonn.de/pctc/mulliken-center/software/dftd4"
+    url = "https://github.com/dftd4/dftd4/archive/refs/tags/v3.2.0.tar.gz"
```

Comment:
Can we use:
```suggestion
    url = "https://github.com/dftd4/dftd4/releases/download/v3.2.0/dftd4-3.2.0.tar.xz"
```
which is an artifact explicitly uploaded by the developers? Or are there issues using it? Asking since, if there are no contraindications, this artifact will maintain a stable sha256 checksum and will not be subject to changes in the way the autogenerated archive is built by Github.